LogInAndLogOut/src/servlets/LogOutServlet.java

```diff
@@ -0,0 +1,22 @@
+package servlets;
+import java.io.IOException;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/LogOut")
+public class LogOutServlet extends HttpServlet  {
+	
+	private static final long serialVersionUID = 1L;
+
+	protected void doGet(HttpServletRequest request,
+		    HttpServletResponse response) throws ServletException, IOException {
```

Comment:
no such exception on this method: ServletException

Reply:
OK